tests/base.py

```diff
@@ -0,0 +1,305 @@
+import os
+from datetime import datetime as dt
+from datetime import timedelta
+
+from tap_tester import connections, menagerie, runner, LOGGER
+from tap_tester.base_suite_tests.base_case import BaseCase
+
+
+class ZoomBase(BaseCase):
+    """
+    Setup expectations for test sub classes.
+    Metadata describing streams.
+
+    A bunch of shared methods that are used in tap-tester tests.
+    Shared tap-specific methods (as needed).
+    """
+
+
+    REPLICATION_KEY_FORMAT = "%Y-%m-%dT00:00:00.000000Z"
+    BOOKMARK_FORMAT = "%Y-%m-%d"
```

Comment:
Do we really need these if every stream is full-table?

Reply:
Done